### ISSUE TITLE
Collab meeting notes

### ISSUE THREAD

**Issue by cjvanlissa** [cjvanlissa/tidySEM#40]
ERC sociology flexible employment with CBS
* Dimitris Pavlopolous
* Licensed software is hard to access in different controlled systems like CBS

None of the packages do all

tidySEM output is still an mxModel object
* They can be customized
* They can be processed downstream

Development goals:
* Mixture models with different items combined 
* Hidden markov models
* Mixtures of markov transition probabilities

Features on user side
* table_fit() does not work for mxMixtures
* Plot AIC BIC
* Categorical data table: table_props()

Passing arguments down to other functions
Allow users to specify bounds etc

**Comment by sda030** [comment]
Regarding the last entry: table_props(), are you considering this for merely pretty outputing a table for e.g. ordinal variables? I would rather Suggest: gtsummary::tbl_summary() and specifically the coming tbl_likert() in the same package (currently in their [development package]https://www.danieldsjoberg.com/bstfun/reference/tbl_likert.html). 
In any case, there are plenty of packages one could import for such use, that already handles html, pdf, word, tibble, etc.

**Comment by cjvanlissa** [comment]
@sda030 no, tidySEM::table_props() reconstructs the probability distribution from thresholds.
